qiskit/circuit/library/standard_gates/rxx.py

```diff
@@ -111,6 +114,39 @@ def _define(self):
 
         self.definition = qc
 
+    def control(
+        self,
+        num_ctrl_qubits: int = 1,
+        label: str | None = None,
+        ctrl_state: str | int | None = None,
+        annotated: bool | None = None,
+    ):
```

Comment:
Would it have been possible to set the default value `annotated=True` without introducing `None`?